core/src/test/java/org/bitcoinj/net/discovery/SeedPeersTest.java

```diff
@@ -41,8 +41,9 @@ public void getPeer_one() throws Exception{
     @Test
     public void getPeer_all() throws Exception{
         SeedPeers seedPeers = new SeedPeers(MAINNET);
-        for (int i = 0; i < MAINNET.getAddrSeeds().length; ++i) {
-            assertThat("Failed on index: "+i, seedPeers.getPeer(), notNullValue());
+        int numPeers = MAINNET.getAddrSeeds().size();
+        for (int i = 0; i < numPeers; i++) {
+            assertThat("Failed on index: " + i, seedPeers.getPeer(), notNullValue());
```

Comment:
This test is using a deprecated constructor and should probably be changed (and maybe minimal tests added for the deprecated methods as we have done in other places)